README.md

```diff
@@ -204,6 +204,25 @@ Steps for using the configuration:
 6. Now the Flameshot entry should appear in the list. Click _Apply_ to apply the changes.
 7. If you want to change the defaults, you can expand the entry, select the appropriate action and modify it as you wish; the process is pretty mush self-explanatory.
 
+#### On Ubuntu and other Gnome based distros
+
+You can easely configure your 'print' keyboard shortcut to be assigned to Flameshot.
+
+1. Open _Settings_ → _Devices_ → _Keyboard_  → _Shortcuts_.
+
+2. Search for 'print', and unbind the screen capture funcion by selecting it, and clicking _backspace_.
+
+3. Scroll down and click on the '+'.
+
+4. On 'Name', name it 'Flameshot' or 'PrintScreen'.
+
+5. Define the command as 'flameshot gui'.
```

Comment:
I think the default behavior of prtscrn is to `flameshot screen` and it would be good to add another one for ctrl or ctrl+shft + prtscrn and map it to `flameshot gui` for doing selections.

Reply:
One more thing, you can write `flameshot config --autostart true` on the terminal to execute automatically when power on PC.
This is exactly the configuration that I use and need.